presto-native-execution/scripts/dockerfiles/README.md

```diff
@@ -0,0 +1,54 @@
+# Prestissimo - Dockerfile build
+
+> 📝 _**Note:** Please post in Presto Slack if you have any questions_
+
+There are two kinds of dockerfiles: 
```

Comment:
```suggestion
There are two kinds of Dockerfiles: 
```
Standard seems to be "Dockerfile" as one word and with initial capitalization. I'll mark these throughout, but please let me know if you have a different understanding of how to commonly refer to this item.